README.md

```diff
@@ -15,21 +15,17 @@ First, add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-futures = "0.2.0"
+futures-preview = "0.3.0-alpha"
 ```
 
 Next, add this to your crate:
 
 ```rust
-extern crate futures;
+extern crate futures-preview as futures;
```

Comment:
A related question: Why does do the `extern crate` statements in the  current master branch (0.2) never use `as ...`? I don't understand why that works...